pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -149,17 +146,16 @@ func configFW() error {
 			glog.Errorf("mellanox-plugin configFW(): failed : %v : %s", err, out)
 			return err
 		}
+		glog.Infof("mellanox-plugin configFW(): %s", out)
 	}
 	return nil
 }
 
-func mstconfigReadData(pciAddress string, attributes ...string) (string, error) {
-	glog.Infof("mellanox-plugin mstconfigReadData(): try to read %s for device %s", attributes, pciAddress)
-
+func mstConfigReadData(pciAddress string) (string, error) {
+	glog.Infof("mellanox-plugin mstConfigReadData(): device %s", pciAddress)
 	args := []string{"-e", "-d", pciAddress, "q"}
-	args = append(args, attributes...)
 	out, err := runCommand("mstconfig", args...)
```

Comment:
Also, can we make the Info print a debug print in runCommand ?

Reply:
good idea